sdc/functions/numpy_like.py

```diff
@@ -967,3 +967,79 @@ def getitem_by_mask_impl(arr, idx):
             return result_data
 
     return getitem_by_mask_impl
+
+
+def skew(a):
+    pass
+
+
+def nanskew(a):
+    pass
+
+
+@sdc_overload(skew)
+def np_skew(arr):
+    if not isinstance(arr, types.Array):
+        return
+
+    def skew_impl(arr):
+        len_val = len(arr)
+        n = 0
+        _sum = 0
```

Comment:
Let's initialize `_sum`, `square_sum` and `cube_sum` with float values `0.`

Reply:
this will lead to errors

Reply:
what kind of errors?